plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/payment-methods-selection/payment-methods-selection.scss

```diff
@@ -0,0 +1,213 @@
+			font-weight: 500;
+			display: inline-flex;
+			gap: 10px;
+			align-items: center;
+		}
+
+		&__item-content {
+			font-size: 13px;
+			font-weight: 400;
+			line-height: 20px;
+			color: $gray-700;
+
+			.woocommerce-woopayments-payment-methods-logos {
+				flex-wrap: wrap;
+			}
+		}
+
+		&__item-after {
+			&__actions {
+				display: flex;
+				gap: $gap;
+				margin-left: $gap;
+
+				.components-base-control {
+					margin-bottom: 0;
+				}
+
+				.components-button {
+					height: 32px;
+				}
+			}
+		}
+
+		&__item-before {
+			img {
+				max-width: 96px;
+				height: 40px;
+				border-radius: 4px;
+			}
+			@media screen and ( max-width: 782px ) {
+				align-items: flex-start;
+
+				img {
+					max-width: 30px;
+					height: 30px;
+					margin-top: 3px;
+				}
+			}
+		}
+
+		&__item-multi-row {
+			display: flex;
+
+			&.multi-row-space {
+				margin-bottom: $gap-large;
+			}
+		}
+	}
+}
+
+.woocommerce-recommended-payment-methods {
+	// Width to fill available space.
+	width: 100%;
+	width: -moz-available;          /* WebKit-based browsers will ignore this. */
+	width: -webkit-fill-available;  /* Mozilla-based browsers will ignore this. */
+	width: fill-available;
+	margin-right: $gap-small + $gap-smallest;
+	display: flex;
+	flex-direction: column;
+	height: calc(100% - 60px); // Account for continue button
+	max-height: 100%; // Ensure it doesn't grow beyond parent
+	overflow: hidden; // Prevent content from spilling out
+	padding: 0 !important; // There is a padding on the parent container that we need to override.
+	border-bottom: none !important;
+
+	@media screen and (max-width: $break-medium) {
+		height: calc(100% - 100px); // Account for small screen content being longer on the page.
+		margin-top: $gap-larger;
+	}
+
+	.woocommerce-layout__header-wrapper {
+		padding: $gap-large 1.5 * $gap-large;
+		border-bottom: 1px solid $gray-200;
+		flex-shrink: 0;
+		background: $white;
+		position: relative;
+		z-index: 1;
+
+		@media screen and (max-width: $break-medium) {
+			padding: $gap $gap-large;
+		}
+
+		.woocommerce-settings-payments-header__title {
+			margin-bottom: $gap-smaller;
+
+			@media screen and (max-width: $break-medium) {
+				padding: 0 !important;
+			}
+		}
+	}
+
+	.woocommerce-recommended-payment-methods__list {
+		flex: 1;
+		min-height: 0;
+		position: relative;
+
+		.settings-payments-methods__container {
+			height: 100%;
+			overflow-y: auto;
+			scrollbar-width: none;
+			-ms-overflow-style: none;
+
+			&::-webkit-scrollbar {
+				display: none;
+			}
+
+			@media screen and (max-width: $break-medium) {
+				padding: $gap;
+			}
+
+			.woocommerce-list {
+				margin-bottom: $gap;
+			}
+		}
+	}
+
+	.woocommerce-recommended-payment-methods__list_footer {
+		display: flex;
+		justify-content: flex-end;
+		padding: $gap-large;
+		background: $white;
+		flex-shrink: 0;
+
+		@media screen and (max-width: $break-medium) {
+			padding: $gap;
+		}
+	}
+}
+
+.settings-payments-methods__show-more {
```

Comment:
I believe this is a duplicate, as the same styles are already applied on line 4.

Reply:
Indeed, I removed the extra styles in [03e847d](https://github.com/woocommerce/woocommerce/pull/56554/commits/03e847ddf5ba958c6d7d9706abeab0607e69439a)